boards/arm/nrf52840_mdk/board.cmake

```diff
@@ -2,3 +2,4 @@
 
 board_runner_args(pyocd "--target=nrf52")
 include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```

Comment:
This wasn't part of your previous revision. Was adding it a mistake? It should either way be in a separate commit.

Reply:
oops, commit gone wrong